Gemfile

```diff
@@ -117,7 +117,7 @@ group :development, :test do
   gem 'awesome_print'
   gem 'dotenv-rails'
   gem 'jquery-rails' # only needed for jasmine-jquery
-  gem 'jasmine', github: 'pivotal/jasmine-gem'
+  gem 'jasmine'
```

Comment:
@akegan jasmine is developed by github. back in the day they had the jasmine gem at https://github.com/pivotal/jasmine-gem, but it's since moved to its own proper rubygems gem at https://rubygems.org/gems/jasmine/, hence this change to draw the gem from rubygems instead.